proxygen_cli/test/command_spec_test.py

```diff
@@ -110,6 +112,30 @@ def test_delete_spec_uat(patch_request, patch_access_token,
         assert path == "/apis/mock-api/spec/uat"
 
 
+@pytest.mark.parametrize("cmd, uat_expected", [
```

Comment:
The test just ensures that the `--uat` option appears in help messages for all relevant sub-commands (e.g. `proxygen spec publish`) **and** the root command `proxygen spec`.